src/RokuDeployOptions.ts

```diff
@@ -174,6 +176,13 @@ export interface RokuDeployOptions {
          */
         formData?: Record<string, any>;
     };
+
+    /**
+     * Should the main library and component libraries be hosted synchronously. This is for when certain libraries like "DCL" require it
+     * if not set the default is false unless one of the libraries requires it
+     * @default true
+     */
+    prepareProjectFilesSynchronously?: boolean;
```

Comment:
This `prepareProjectFilesSynchronously` doesn't belong in roku-deploy. roku-debug needs the setting but it shouldn't leak into this project as it's not used anywhere in here.